etc/scripts/generate-github-test-data.py

```diff
@@ -0,0 +1,140 @@
+# HOW TO USE ME?
+# `python3 /Users/jkay/repos/github-for-jira/etc/scripts/generate-github-test-data.py
+# Generate a personal access token from GitHub(PAT)
+# replace ACCESS_TOKEN with the PAT
+# replace USERNAME with GitHub username
+#
+import requests
```

Comment:
Nit: This makes raw API requests right? Just had a geez and it looks like there are a couple of libraries that specifically designed for GitHub interactions: PyGithub and github3.py. Never used either of them, and this library is fine to stick with, just a thought.

Reply:
Yeah, probably would have been a good idea was just powering through at path of least resistance tbh

Reply:
Then along comes Rachelle, all the resistance :wink_: